includes/internal/emails/class-email-seeder-data.php

```diff
@@ -69,7 +69,7 @@ public function get_email_data(): array {
 			],
 			'student_starts_course'       => [
 				'types'       => [ 'teacher' ],
-				'subject'     => __( '[student:displayname] started [course:name]', 'sensei-lms' ),
+				'subject'     => __( '[student:displayname] enrolled in [course:name]', 'sensei-lms' ),
 				'description' => __( 'Course Started', 'sensei-lms' ),
```

Comment:
What do you think about updating the description for this email?

Reply:
Thanks! Updated here https://github.com/Automattic/sensei/pull/7661/commits/9336e3cd4311756a286dd4644a72918ae156395b